docs/deployment/minikube.md

```diff
@@ -58,7 +58,9 @@ Kong is accesssible. This will be used to actually send reqeusts into the
 Kubernetes cluster.
 
 ```bash
-$ export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)
+$ export PROXY_IP=$(minikube service -n kong kong-kong-proxy --url | head -1)
```

Comment:
This is incorrect. It should be `kong-proxy` and not `kong-kong-proxy`.

Reply:
To my test, it was `kong-kong-proxy`. Would it be because I installed it by helm chart with `kong` as a release name?

Reply:
> So if release name is foo, then the service name would become foo-kong-proxy?

Yes.

Isn't this line meant for manifest installation and not Helm?

Reply:
Ah, yes then kong-proxy should be used.